int32 user_id
string username
